### DISCUSSION TITLE
Article Layout Builder / Missing Intro Image

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#794] [General]
I miss the selection for Intro Image in the system elements. If I select Full Image instead, this doesn't really get me where I want to go, as only a section of the image is displayed and not the complete image.
Best regards

![system_elements](https://github.com/user-attachments/assets/23c4bf8b-fb96-490e-baaf-63c076f2e345)

**Comment by jam1974** [comment]
The intro image is only for the view in the category and is not displayed in the article. It is no different if you create the article with joomla tools.

**Comment by Banuk1616** [comment]
@jam1974 
Yes, I know. I also mean the view in the category. Like here in the demo (article Ultimate Northern Greece):
[https://demo.astroidframe.work/article-layouts/tours-example](url)
Shouldn't the intro image be there?
Best regards

**Reply by sonvnn** [reply]
Intro Image displays on Category View only. Link you mention above is Category view. 
Full Image displays on Detail Article only.
Article Layout is only used for Detailed Article. That's why only Full Image element display on Article Layout Builder

Best Regards,
Sonny

**Comment by Banuk1616** [comment]
@sonvnn 
O.K, Thanks for Info.
Best regards

**Reply by sonvnn** [reply]
You're welcome :)
